src/pages/UpdateSuccess/UpdateSuccess.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+// import propTypes from 'prop-types';
+import PromoHeader from '../../components/PromoHeader/PromoHeader';
+import Cards from '../../components/Cards/Cards';
+
+class Success extends Component {
+  render() {
+    /*    const hasState = this.props.location.state;
+    const firstName = this.props.location.state.firstname;
+    const giftAidChoice = this.props.location.state.giftAidChoice; */
+
+    // HARDCODED DEBUG VALUES!
+    const hasState = true;
+    const firstName = 'Andy';
+    let giftAidChoice = 0;
+    giftAidChoice = 1;
+
```

Comment:
not sure if you're finished, so just a reminder that these are still there. (And the commented out properties at the bottom.)

Reply:
Will be doing the success stuff in the next PR, as discussed